.github/workflows/run.yml

```diff
@@ -17,7 +17,7 @@ jobs:
 
       - name: Build Saxpy Workspace
         run: |
-          ./bin/benchpark setup saxpy/openmp x86 workspace/
+          ./bin/benchpark setup saxpy/openmp x86_64 workspace/
```

Comment:
Are we going to have to specify these entire names from the CLI? We're going to need some sort of tab completion eventually.

Reply:
Yes.  I also hope making suites will help.